src/wallet/walletdb.cpp

```diff
@@ -218,8 +218,10 @@ CAmount CWalletDB::GetAccountCreditDebit(const string &strAccount)
     ListAccountCreditDebit(strAccount, entries);
 
```

Comment:
like for the other case wher that happened, remove also `#include <boost/foreach.hpp>`

Reply:
done for all the above